docs/tutorials/mongoid-transactions.txt

```diff
@@ -0,0 +1,115 @@
+=========
+Transactions
+=========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Version 4.0 of the MongoDB server introduces
+`multi-document transactions <https://docs.mongodb.com/manual/core/transactions/>`_.
+(Updates to multiple fields within a single document are atomic in all
+versions of MongoDB). Mongoid version 7.1.0 adds support for transactions.
```

Comment:
Going to our earlier discussion about backporting transactions to 6.x, it is my impression that 6.x should support transactions with 2.6.0+ driver.

Reply:
Good point! I think we should probably at some explicit testing for this, but I'm fine with documenting it for now.